src/components/organize/journeys/JourneyPerson.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box } from '@material-ui/core';
+import { Close } from '@material-ui/icons';
+import { grey } from '@material-ui/core/colors';
+import { useState } from 'react';
+
+import ZetkinPerson from 'components/ZetkinPerson';
+import { ZetkinPerson as ZetkinPersonType } from 'types/zetkin';
+
+const JourneyPerson = ({
+  person,
+  onRemove,
+}: {
+  onRemove: (person: ZetkinPersonType) => void;
+  person: ZetkinPersonType;
+}): JSX.Element => {
+  const [hover, setHover] = useState<boolean>(false);
+  return (
+    <Box
+      alignItems="center"
+      bgcolor={hover ? grey[200] : ''}
+      display="flex"
+      justifyContent="space-between"
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+      p={1}
+    >
+      <ZetkinPerson
```

Comment:
I just realized, we need to add the person hover card to this component!

Reply:
Is this a) not a problem, b) worth a separate issue, or c) something worth spending time/energy on in this PR?